voskcli/transcribe.py

```diff
@@ -67,6 +75,21 @@ def model_path(model):
 
 
 def write_captions_paragraph(vtt, paragraph):
+    '''
+    Append time-coded paragraphs to vtt file.
+    
+    :param vtt: Empty or partially filled WebVTT file to write transcriptions to
+    :type vtt: WebVTT
```

Comment:
`print(type(vtt))` returns `<class webvtt.webvtt.WebVTT>`

Reply:
I would probably call this something like `WebVTT object` to make sure people won't mistake it for a binary WebVTT file, but other than that, this should work.